fds/services/fds_service.py

```diff
@@ -97,3 +97,37 @@ def commit(self, message: str, yes: bool):
             self.printer.error(str(e))
             self.printer.error("Git commit failed to execute")
             sys.exit(1)
+
+    def push(self, git_remote: str, dvc_remote: str, ref: str = None):
+        """
+        fds push
+        """
+        try:
+            self.dvc_service.push(remote=dvc_remote)
+            self.printer.warn("Successfully pushed to DVC remote")
+        except Exception as e:
+            self.printer.error(str(e))
+            self.printer.error("DVC push failed to execute")
+            sys.exit(1)
+        try:
+            self.git_service.push(remote=git_remote, ref=ref)
+            self.printer.success("Successfully pushed to Git remote")
+        except Exception as e:
+            self.printer.error(str(e))
+            self.printer.error("Git push failed to execute")
+            sys.exit(1)
```

Comment:
Might want to switch the order to git push first - dvc push can take a long time

Reply:
Ok